### ISSUE TITLE
Add option for coverage data file

### ISSUE THREAD

**Issue by zageyiff** [cmheisel/nose-xcover#24]
Before anything else I want to thank for this great plugin.

This is not an issue, but more a request.
To add the option of where to generate the coverage data file. As of now the file is generated at the same level of the setup.py or cwd in a file called '.coverage'

Thanks

**Comment by wojtossfm** [comment]
Not sure if I understand the issue correctly but isn't this already possible by using the `--xcoverage-file=<path>` option? Or alternatively you could use `--xcoverage-to-stdout` and redirect to wherever you want. Please maybe rephrase if I'm mistaken.
